.github/CODEOWNERS

```diff
@@ -44,6 +44,7 @@
 /jobs/**/*.py                          @GoogleCloudPlatform/python-samples-reviewers
 /kubernetes_engine/**/*.py             @GoogleCloudPlatform/python-samples-reviewers
 /kubernetes_engine/django_tutorial/**/*.py @glasnt @GoogleCloudPlatform/python-samples-reviewers
+/media-cdn/**/*.py                     @justin-mp @msampathkumar @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
should this be an underscore? media_cdn instead of a dash?

Reply:
(I know the directories are media-cdn but I'm wondering if the preference is to not use - in directories and go with the underscore)

Reply:
Hi @iennae, Thank you for you suggestion.

I tried to follow the pattern I saw for [data-science-onramp](https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/data-science-onramp) 

Also cloud docs for `media-cnd`, at [https://cloud.google.com/**media-cdn**/docs](https://cloud.google.com/media-cdn/docs) following the same pattern. So `media-cdn` felt the right way to go.

Please suggest otherwise. I am open to update it as suggested.

Reply:
if i had to do it over, I'd do data_science_onramp instead - go with the underscore! 😁

Reply:
@iennae, @leahecole Thank you 😁